scripts/populate_institutions.py

```diff
@@ -83,7 +83,7 @@ def main(env):
             {
                 '_id': 'usc',
                 'name': 'University of Southern California',
-                'description': 'Projects must abide by <a href="http://policy.usc.edu/info-security/">USC\'s Information Security Policy</a>. Data stored for human subject research repositories must abide by <a href="http://policy.usc.edu/biorepositories/">USC\'s Biorepository Policy</a>. The OSF may not be used for storage of Personal Health Information that is subject to <a href="http://policy.usc.edu/hipaa/">HIPPA regulations</a>.',
+                'description': '<h5>Projects must abide by <a href="http://policy.usc.edu/info-security/">USC\'s Information Security Policy</a>. Data stored for human subject research repositories must abide by <a href="http://policy.usc.edu/biorepositories/">USC\'s Biorepository Policy</a>. The OSF may not be used for storage of Personal Health Information that is subject to <a href="http://policy.usc.edu/hipaa/">HIPPA regulations</a>.</h5>',
```

Comment:
for now those make sense as h5, however, future ones might have more/less text. As suggested by Michael, since we are allowing html, it would make sense to decouple formatting the description   from the template.